pkgs/servers/gobetween/default.nix

```
@@ -0,0 +1,33 @@
+{ buildGo112Module, fetchFromGitHub, lib, enableStatic ? false }:
+
+buildGo112Module rec {
+  pname = "gobetween";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "yyyar";
+    repo = "gobetween";
+    rev = "7d8a7368a535641df273c6b962750c1113d0c6b0";
+    sha256 = "628e9a3f37aa12f3c54ef7b81a707ea732179a0aa7bb621b8cd833535f27546e";
+  };
+  modSha256 =
```

should be one line, and please put a space between that and src
```suggestion

  modSha256 =
```

---

added space. long line is split by [nixfmt](https://github.com/serokell/nixfmt), it does look odd, but i'm trying to apply nixfmt as much as possible. I can change if it's a problem.